x-pack/plugins/security/server/audit/audit_events.test.ts

```diff
@@ -300,6 +301,35 @@ describe('#userLoginEvent', () => {
   });
 });
 
+describe('#userLogoutEvent', () => {
+  test('creates event with `success` outcome', () => {
```

Comment:
__nit:__ if we intentionally support `undefined` for `username` let's maybe add a test for this case as well (to make sure nothing breaks badly)?